lib/helper/WebDriver.js

```diff
@@ -1738,6 +1738,21 @@ class WebDriver extends Helper {
     }, aSec * 1000, `element (${locator}) still not present on page after ${aSec} sec`);
   }
 
+  /**
+   * {{> waitForClickable }}
+   * {{ react }}
```

Comment:
```suggestion
```
React is not supported. You use `$$`. And I don't think react locators should work here